pyoozie/builder.py

```diff
@@ -30,7 +30,7 @@ class WorkflowBuilder(object):
 
     def __init__(self, name):
         # Initially, let's just use a static template and only one action payload and one action on error
-        self._name = _validate_name(name)
+        self._name = tags._validate_name(name)
```

Comment:
I think that you're right; it's probably useful for someone developing their own name-generator to know ahead of time which names would be unacceptable (https://github.com/Shopify/pyoozie/issues/18). Thought for a future PR.